CRAMv3.tex

```diff
@@ -477,15 +477,17 @@ \section{\textbf{Container header structure}}
 \hline
 \end{tabular}
 
-\subsection{\textbf{CRAM header in the first container}}
+\subsection{\textbf{BAM header in the first container}}
 
-The first container in the CRAM file contains the BAM header in an optionally
-compressed block. Only gzip is allowed as compression method for this block. BAM
-header is terminated with \textbackslash{}0 byte and any extra bytes in the
-block can be used to expand the BAM header. For example when updating @SQ
-records additional space may be required for the BAM header. It is recommended
-to reserve 50\% more space in the CRAM header block than it is required by the
-BAM header.
+The first container in a CRAM file contains the BAM header in an optionally
+compressed block. Only gzip is allowed as compression method for this block.
+The BAM header container does not include a compression header block.
+
+It is recommended to reserve 50\% more space in the BAM header container than
+is required for the BAM header by optionally padding the container with a second
+raw block consisting of all zeroes. This can be used to subsequently expand the header
+container in place, such as when updating @SQ records, while preserving the relative
```

Comment:
"while preserving the *absolute* offsets of all subsequent containers".

Reply:
Oh yeah...Done.